.github/workflows/tests.yml

```diff
@@ -144,20 +144,21 @@ jobs:
           path: src/${{ github.event.repository.name }}
       - name: Install dependencies and build
         shell: bash
+        # Downgrade pytest to version 8.0.2 due to issue with required parameter 'consider_namespace_packages' not satisfied with version 8.1.1: https://github.com/ros2/launch/issues/765
         run: |
           apt-get update
-          apt-get -y install python3-coverage python3-pip python3-pytest python3-pytest-asyncio ros-foxy-rclpy unzip
-          python3 -m pip install nats-py influxdb influxdb_client mcap-ros2-support pathlib pytest-env pytest-asyncio aiounittest mysql-connector-python
-          apt install -y apt-utils python3-pybind11 ros-foxy-test-msgs python3-colcon-common-extensions
-          git clone https://github.com/ros2/rosbag2.git --branch foxy-future src/rosbag2
-          ls -la
-          source /opt/ros/foxy/setup.bash
-          colcon build --packages-up-to rosbag2_processing_service
+          apt-get -y install python3-coverage python3-pip python3-pytest python3-pytest-asyncio
+          pip3 install pytest==8.0.2
```

Comment:
Can we add a comment here about why we are using this version?

Reply:
added